apps/workflows/jobs/refill-daily.ts

```diff
@@ -44,36 +67,44 @@ client.defineJob({
             remaining: key.refillAmount,
             lastRefillAt: new Date(),
           })
-          .where(eq(schema.keys.id, key.id));
+          .where(eq(schema.keys.id, key.id))
```

Comment:
this prevents the call from throwing, which prevents trigger from knowing it failed, and you're not logging with trigger's logger either. Do we even know when this goes wrong in prod?